### ISSUE TITLE
Plugin failed to load: Api version is not compatible

### ISSUE THREAD

**Issue by RebirthGaming** [Archomeda/TS3-GW2-plugin#1]
Plugin installed just fine, but when viewing the plugin in Teamspeak via Settings > Plugins, the error displayed in Description is:  

Plugin failed to load:
Api version is not compatible

Your API version is 19. API version 20 is required. 

Teamspeak3 version 3.0.16, Qt Version 5.2.1

**Comment by Archomeda** [comment]
Yeah, apparently TS3 updated the API version some time ago with a newer TS3 version. Because I updated this 2 years ago for the last time, I'm unsure whether or not I'll maintain it anymore. You can try it yourself to change [the API version](https://github.com/Archomeda/TS3-GW2-plugin/blob/cf241efa84d3d8ab4a90f91e1d8005d70d1b93fd/src/globals.h#L19) and compile it to see if it works. You might need to update the dependencies from TS3 as well.

**Comment by bluelovers** [comment]
hope can update this plugin
